src/perf/bin/appmain.cpp

```diff
@@ -41,7 +45,24 @@ QuicUserMain(
     QUIC_EVENT StopEvent;
     QuicEventInitialize(&StopEvent, true, false);
 
-    QUIC_STATUS Status = QuicMainStart(argc, argv, &StopEvent, SelfSignedConfig);
+    uint8_t ServerMode = 0;
+    TryGetValue(argc, argv, "ServerMode", &ServerMode);
+
+    QUIC_STATUS Status;
+    QUIC_DATAPATH* Datapath;
+    QUIC_DATAPATH_BINDING* Binding;
+
+    if (ServerMode) {
+        QuicDataPathInitialize(256, DatapathReceiveUserMode, DatapathUnreachable, &Datapath);
```

Comment:
You should just be able to pass 0 instead of 256. You don't need any callback context.